### PR TITLE
Re-Run of F-24B/C/D after bug fix

### PR DESCRIPTION
**Pull request by Jyoti-cs**
Signed-off-by: Jyoti Baral <jyoti.baral@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: tools/setup_update/setup_entry.json [hunk 1]
```diff
@@ -1,125 +1,103 @@
-{
```
Comment (sonalk0209) [inline tools/setup_update/setup_entry.json:1]: why do u want to add this file for review?

File: comptests/ha/test_failure_handling_pod_failure.py [hunk 2]
```diff
@@ -108,11 +110,31 @@ def teardown_method(self):
...
+            assert_utils.assert_equal(data1, 'monitor',
+                                      f"Source of {generation_id_list[index]} is not from monitor")
+            assert_utils.assert_equal(data2, 'node',
+                                      f"Resource of {generation_id_list[index]} is not node")
+            assert_utils.assert_equal(data3, 'failed',
+                                      f"Resource status of {generation_id_list[index]} "
+                                      "is not failed")
+
+    @pytest.mark.skip(reason="CORTX-28823")
     @pytest.mark.ha
```
Comment (KASwami) [inline comptests/ha/test_failure_handling_pod_failure.py:138]: Please add correct marker comp_ha

File: comptests/ha/test_failure_handling_pod_failure.py [hunk 3]
```diff
@@ -193,39 +216,27 @@ def test_delete_pod_replicaset_down(self):
...
-                break
-        cmd_halog = "tail -5 /mnt/fs-local-volume/local-path-provisioner/" \
-                    + hapvc + "/log/ha/*/health_monitor.log | grep 'to component hare'"
-        output = node_obj.execute_cmd(cmd_halog)
-        LOGGER.info("Events output :%s", output)
-
-        # TODO: Pod failure alert verification - EOS-28560
-        # TODO: Correct event and contents published to component Hare - EOS-28560
+        LOGGER.info("Step 2: Check the node status.")
+        time.sleep(HA_CFG["common_params"]["30sec_delay"])
```
Comment (Pradeep-B-Thangaraj) [inline comptests/ha/test_failure_handling_pod_failure.py:220]: Why this delay is needed?

Comment (Pradeep-B-Thangaraj) [inline comptests/ha/test_failure_handling_pod_failure.py:220]: Why this delay is needed?

File: comptests/ha/test_failure_handling_pod_failure.py [hunk 4]
```diff
@@ -257,39 +269,27 @@ def test_delete_pod_deployment(self):
...
```
Comment (Pradeep-B-Thangaraj) [inline comptests/ha/test_failure_handling_pod_failure.py:273]: Here also why delay is needed?

File: comptests/ha/test_failure_handling_pod_failure.py [hunk 5]
```diff
@@ -257,39 +269,27 @@ def test_delete_pod_deployment(self):
...
+        resp = self.node_master_list[0].get_all_pods(pod_prefix=common_const.POD_NAME_PREFIX)
+        after_del = len(resp)
+        LOGGER.info("After delete %s", after_del)
+        LOGGER.info("Before delete %s", before_del)
+        assert_utils.assert_not_equal(after_del, before_del, "Data pod didn't gets deleted")
+        LOGGER.info("Step 2: Node status shows with the deleted pod")
```
Comment (Pradeep-B-Thangaraj) [inline comptests/ha/test_failure_handling_pod_failure.py:279]: Change logger statement properly.

**Comment by Pradeep-B-Thangaraj** [comment]
@Jyoti-cs As these are already in was not able to give comments, so added here. Check and change accordingly.
Line 82 to 94 - We can make use of libs as we do in other scripts.
Line 319 - Check if we need logger.

File: comptests/ha/test_failure_handling_pod_failure.py [hunk 6]
```diff
@@ -108,12 +110,32 @@ def teardown_method(self):
...
-            assert_utils.assert_true(resp[0], resp[1])
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Cleanup: Cluster status checked successfully")
         LOGGER.info("Done: Teardown completed.")
 
-    @pytest.mark.ha
+    @staticmethod
+    def get_node_status_log(resp_dict):
+        """
+        This is a local method to verify the status and get node and resource ids
```
Comment (Pradeep-B-Thangaraj) [inline comptests/ha/test_failure_handling_pod_failure.py:120]: This doc string is mismatching.

**Review by KASwami — APPROVED** [review]
Looks good

**Review by Pradeep-B-Thangaraj — APPROVED** [review]
Looks good!

**Review by sonalk0209 — APPROVED** [review]
Approving for merge.

**Comment by sonalk0209** [comment]
@Jyoti-cs add collections log

**Comment by Jyoti-cs** [comment]
[terminal.log](https://github.com/Seagate/cortx-test/files/8321203/terminal.log)
@sonalk0209 , please find the collection logs attached.

**Pull request merged**